server/middleware/form/process.js

```diff
@@ -97,6 +97,23 @@ const isUnacceptedDataFields = (field) => {
         isFieldComponentOfType(field, 'somu-list');
 };
 
+function checkValidationCondition(validator, caseData) {
+    if (Object.prototype.hasOwnProperty.call(validator, 'condition') && caseData[validator.condition.fieldName] !== validator.condition.value) {
```

Comment:
Would it be simpler to inverse this? Return true only in this case and default to false? Unsure if that has side effects though...